packages/atomic/src/components/commerce/facets/atomic-commerce-numeric-facet/atomic-commerce-numeric-facet.tsx

```diff
@@ -0,0 +1,277 @@
+    this.range = {start: detail.start, end: detail.end};
+  }
+
+  public render() {
+    const {
+      bindings: {i18n},
+    } = this;
+    const {firstSearchExecuted, hasError} = this.summary.state;
+    return (
+      <FacetGuard
+        enabled={true}
+        firstSearchExecuted={firstSearchExecuted}
+        hasError={hasError}
+        hasResults={this.shouldRenderFacet}
+      >
+        {firstSearchExecuted ? (
+          <FacetContainer>
+            <FacetHeader
+              i18n={i18n}
+              label={this.displayName}
+              onClearFilters={() => {
+                this.focusTarget.focusAfterSearch();
+                this.facet.deselectAll();
+                this.facet.setRanges([]);
+              }}
+              numberOfActiveValues={this.numberOfSelectedValues}
+              isCollapsed={this.isCollapsed}
+              headingLevel={0}
+              onToggleCollapse={() => (this.isCollapsed = !this.isCollapsed)}
+              headerRef={(el) => this.focusTarget.setTarget(el)}
+            />
+            {!this.isCollapsed && [
+              this.shouldRenderValues && this.renderValues(),
+              this.shouldRenderInput && (
+                <atomic-commerce-facet-number-input
+                  bindings={this.bindings}
+                  label={this.displayName}
+                  facet={this.facet}
+                  range={this.range}
+                ></atomic-commerce-facet-number-input>
+              ),
+            ]}
+          </FacetContainer>
+        ) : (
+          <FacetPlaceholder isCollapsed={this.isCollapsed} numberOfValues={8} />
+        )}
+      </FacetGuard>
+    );
+  }
+
+  private renderValues() {
+    const {
+      manualRanges,
+      displayName,
+      bindings: {
+        i18n,
+        engine: {logger},
+      },
+      formatter,
+    } = this;
+
+    return (
+      <NumericFacetValuesContainer i18n={i18n} label={displayName}>
+        {this.valuesToRender.map((value) => (
+          <NumericFacetValueLink
+            formatter={formatter}
+            displayValuesAs={'checkbox'}
+            facetValue={value}
+            field={this.state.field}
+            i18n={i18n}
+            logger={logger}
+            manualRanges={manualRanges}
+            onClick={() => this.facet!.toggleSelect(value)}
+          />
+        ))}
+      </NumericFacetValuesContainer>
+    );
+  }
+
+  private get state() {
+    return this.facet.state;
+  }
+
+  private get displayName() {
+    return this.state.displayName || 'no-label';
+  }
+
+  private get numberOfSelectedValues() {
+    if (this.range) {
+      return 1;
+    }
+
+    return (
+      this.facet?.state.values.filter(({state}) => state === 'selected')
+        .length || 0
+    );
+  }
+
+  private get hasInputRange() {
+    return !!this.range;
+  }
+
+  private get shouldRenderValues() {
+    return !this.hasInputRange && !!this.valuesToRender.length;
+  }
+
+  private get valuesToRender() {
```

Comment:
It could also be worthwhile to add this information in the numeric facet controller state